### DISCUSSION TITLE
Package manager

### DISCUSSION THREAD

**Discussion by jasal82** [koto-lang/koto#400] [Show and tell]
Hi,

just wanted to let you know that I developed a package manager for my application which can resolve a dependency graph and uses de-centralized infrastructure (Git repositories) as package and index storage. It uses koto as embedded language and makes koto scripts from the dependencies available via the prelude. Maybe this could be developed into a full-fledged package manager for koto itself. Here is a link to my project:

https://github.com/jasal82/metabuild

**Comment by irh** [comment]
Thanks @jasal82, that's great to see 👍 Do you have an example of the kind of script that you're using with the system? I don't see any koto files in the repo.

**Comment by jasal82** [comment]
I added some examples at the bottom of readme, it's still work in progress.
